src/components/SignIn/index.jsx

```diff
@@ -0,0 +1,191 @@
+
+import * as firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import db from '../../firebaseConfig';
+import React, { Component } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+
+class SignIn extends Component {
+ 
+  handleSubmit = (e) => {
+    debugger
+    e.preventDefault();
+    firebase.auth().signInWithEmailAndPassword(e.target.elements[0].value, e.target.elements[1].value).catch(function(error) {
+        // Handle Errors here.
+        var errorCode = error.code;
+        var errorMessage = error.message;
+        console.log(errorCode, errorMessage);
+      });
+    console.log(this.state);
+  }
+  render() {
+    return (
+      <div className="container">
+        <Form onSubmit={this.handleSubmit}>
+          <Form.Group as={Row} controlId="formBasicEmail">
+            <Form.Label column sm={2}>Email address</Form.Label>
+            <Col sm={10}>
+            <Form.Control type="email" 
+            placeholder="Enter email" />
+            <Form.Text className="text-muted">
+              We'll never share your email with anyone else.
+            </Form.Text>
+            </Col>
+          </Form.Group>
+
+          <Form.Group as={Row} controlId="formBasicPassword">
+            <Form.Label>Password</Form.Label>
+            <Form.Control type="password" placeholder="Password" />
+          </Form.Group>
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </Form>
+      </div>
+    )
+  }
+}
+
+export default SignIn;
+
+/*
```

Comment:
I think you forgot to delete the comments

Reply:
> I think you forgot to delete the comments

oops! thank you